README.md

```diff
@@ -23,10 +23,10 @@ Build binaries for all supported platforms:
 $ make -f docker.Makefile cross
 ```
 
-Run all linting:
+Run all linting and shellcheck validation:
 
 ```
-$ make -f docker.Makefile lint
+$ make -f docker.Makefile lint shellcheck
```

Comment:
We can mention the `help` make target in the readme (to see a list of all make options, type make ......help)